src/resources/companyResources.ts

```diff
@@ -0,0 +1,254 @@
+import { ResourceIF } from '@/interfaces'
+
+export const CompanyResources: Array<ResourceIF> = [
+  {
+    entityType: 'BEN',
+    displayName: 'BC Benefit Company',
+    title: 'Benefit Company Statement',
+    description: `This company is a benefit company and, as such, has purposes that include conducting its business
+          in a responsible and sustainable manner and promoting one or more public benefits.`,
+    statement: '',
+    nameRequestType: 'BC',
+    directors: {
+      countMinimum: 1
```

Comment:
Example: if `countMinimum` is not used for this entity type then we write this:
```
   countMinimum: 0 // not used by this entity type
```

Reply:
In this scenario we don't have any rules that aren't applicable to the current entities we are trying to set a baseline for, but will 100% follow this approach when we do!

After looking at your RFC, i like the idea to apply the rules within here as well! Will do that 👍